src/Request.js

```diff
@@ -1,6 +1,7 @@
 class Request {
 	constructor(payload) {
 		this.baseUrl = "";
+		this.urlParams = "";
```

Comment:
I'm still a bit hesitant on the naming of this since it's so similar to URLSearchParams which is a reference to the query string params, but hopefully since it's missing the word "search" we wont' confuse our future selves when seeing this

Reply:
What if we called it `baseUrlParams`?

Reply:
I like that idea! Updating now really quick